src/Templates/src/templates/maui-blazor/.template.config/template.json

```diff
@@ -85,6 +85,24 @@
           "toLower": true
         }
       },
+      "nameToAppId": {
+        "type": "generated",
+        "generator": "regex",
+        "dataType": "string",
+        "parameters": {
+          "source": "nameToLower",
+          "steps": [
+            {
+              "regex": "\\s",
+              "replacement": "_"
+            },
+            {
+              "regex": "[^a-z0-9_]",
```

Comment:
This is a very limited set of 37 characters. Is it required to be so limited? Nothing non-English allowed? (If that's the case, it's fine, but I want to make sure it's not overly restricted.) Also what about `.` (dot/period) characters, which I think _are_ allowed?

Reply:
Yeah I just realized that dots are allowed. Reading from the error that the user reported, the right regex would be `[-._a-z0-9]`. It has to be alphanumeric though, otherwise app stores won't accept it: https://developer.android.com/build/configure-app-module#:~:text=Although%20the%20application%20ID%20looks,zA%2DZ0%2D9_%5D.